docs/releases/index.html

```diff
@@ -594,46 +594,46 @@ <h1>Releases</h1>
-<td><a href="https://s3.amazonaws.com/xfra-layer0/release/v0.10.3/layer0_v0.10.3_windows.zip">Windows</a></td>
+<td>v0.9.0</td>
+<td><a href="https://github.com/quintilesims/layer0/releases/download/v0.9.0/macOS.zip">macOS</a></td>
+<td><a href="https://github.com/quintilesims/layer0/releases/download/v0.9.0/Linux.zip">Linux</a></td>
+<td><a href="https://github.com/quintilesims/layer0/releases/download/v0.9.0/Windows.zip">Windows</a></td>
 </tr>
 <tr>
-<td>v0.10.2</td>
-<td><a href="https://s3.amazonaws.com/xfra-layer0/release/v0.10.2/layer0_v0.10.2_darwin.zip">macOS</a></td>
-<td><a href="https://s3.amazonaws.com/xfra-layer0/release/v0.10.2/layer0_v0.10.2_linux.zip">Linux</a></td>
-<td><a href="https://s3.amazonaws.com/xfra-layer0/release/v0.10.2/layer0_v0.10.2_windows.zip">Windows</a></td>
+<td>v0.10.0</td>
+<td><a href="https://github.com/quintilesims/layer0/releases/download/v0.10.0/macOS.zip">macOS</a></td>
+<td><a href="https://github.com/quintilesims/layer0/releases/download/v0.10.0/Linux.zip">Linux</a></td>
+<td><a href="https://github.com/quintilesims/layer0/releases/download/v0.10.0/Windows.zip">Windows</a></td>
 </tr>
 <tr>
 <td>v0.10.1</td>
-<td><a href="https://s3.amazonaws.com/xfra-layer0/release/v0.10.1/layer0_v0.10.1_darwin.zip">macOS</a></td>
-<td><a href="https://s3.amazonaws.com/xfra-layer0/release/v0.10.1/layer0_v0.10.1_linux.zip">Linux</a></td>
-<td><a href="https://s3.amazonaws.com/xfra-layer0/release/v0.10.1/layer0_v0.10.1_windows.zip">Windows</a></td>
+<td><a href="https://github.com/quintilesims/layer0/releases/download/v0.10.1/macOS.zip">macOS</a></td>
+<td><a href="https://github.com/quintilesims/layer0/releases/download/v0.10.1/Linux.zip">Linux</a></td>
+<td><a href="https://github.com/quintilesims/layer0/releases/download/v0.10.1/Windows.zip">Windows</a></td>
 </tr>
 <tr>
-<td>v0.10.0</td>
-<td><a href="https://s3.amazonaws.com/xfra-layer0/release/v0.10.0/layer0_v0.10.0_darwin.zip">macOS</a></td>
-<td><a href="https://s3.amazonaws.com/xfra-layer0/release/v0.10.0/layer0_v0.10.0_linux.zip">Linux</a></td>
-<td><a href="https://s3.amazonaws.com/xfra-layer0/release/v0.10.0/layer0_v0.10.0_windows.zip">Windows</a></td>
+<td>v0.10.2</td>
+<td><a href="https://github.com/quintilesims/layer0/releases/download/v0.10.2/macOS.zip">macOS</a></td>
+<td><a href="https://github.com/quintilesims/layer0/releases/download/v0.10.2/Linux.zip">Linux</a></td>
+<td><a href="https://github.com/quintilesims/layer0/releases/download/v0.10.2/Windows.zip">Windows</a></td>
 </tr>
 <tr>
-<td>v0.9.0</td>
-<td><a href="https://s3.amazonaws.com/xfra-layer0/release/v0.9.0/layer0_v0.9.0_darwin.zip">macOS</a></td>
-<td><a href="https://s3.amazonaws.com/xfra-layer0/release/v0.9.0/layer0_v0.9.0_linux.zip">Linux</a></td>
-<td><a href="https://s3.amazonaws.com/xfra-layer0/release/v0.9.0/layer0_v0.9.0_windows.zip">Windows</a></td>
+<td>v0.10.3</td>
+<td><a href="https://github.com/quintilesims/layer0/releases/download/v0.10.3/macOS">macOS</a></td>
```

Comment:
I'm not sure how this happened if it was autogenerated, but somehow the .zip extension is missing from this link.

Reply:
Good catch. I'm not sure what happened I'll check that the autogen is still working